File: RP1/DesafioROI.cpp
#include <iostream>
using namespace std;

int main() {
    double lucro, investimento;

    cout << "Digite o lucro: ";
    cin >> lucro;

    cout << "Digite o investimento: ";
    cin >> investimento;

    if (investimento == 0) {
        cout << "Erro: investimento não pode ser zero." << endl;
    } else {
        double roi = (lucro - investimento) / investimento;
        cout << "ROI: " << roi << endl;
    }

    return 0;
}   

File: RP1/DesafioROI.java
import java.util.Scanner;

public class DesafioROI {
    public static void main(String[] args) {

        try (Scanner sc = new Scanner(System.in)) {

            double lucro, investimento;

            System.out.print("Digite o lucro: ");
            lucro = sc.nextDouble();

            System.out.print("Digite o investimento: ");
            investimento = sc.nextDouble();

            if (investimento == 0) {
                System.out.println("Erro: investimento não pode ser zero.");
            } else {
                double roi = (lucro - investimento) / investimento;
                System.out.println("ROI: " + roi);
            }
        }
    }
}